src/PostgREST/Config.hs

```diff
@@ -359,17 +359,17 @@ parser optPath env dbSettings =
 
     coerceInt :: (Read i, Integral i) => C.Value -> Maybe i
     coerceInt (C.Number x) = rightToMaybe $ floatingOrInteger x
-    coerceInt (C.String x) = readMaybe $ toS x
+    coerceInt (C.String x) = readMaybe (toS x :: [Char])
```

Comment:
I think we'd typically call this `String` instead of `[Char]`?

Reply:
> An alternative (slightly preferable looking forward?) would be a diff like the following:

Yes, I think that's the way to go. I modified the protolude version and the affected lines.